src/samples/simple-freedom-chat/freedom-module.ts

```diff
@@ -68,18 +68,18 @@ function makePeerConnection(name:string) {
   return pc;
 }
 
-var a :PeerConnection<SignallingMessage> = makePeerConnection('A');
-var b :PeerConnection<SignallingMessage> = makePeerConnection('B')
+var a :PeerConnection<signals.Message> = makePeerConnection('A');
+var b :PeerConnection<signals.Message> = makePeerConnection('B')
 
 // Connect the two signalling channels. Normally, these messages would be sent
 // over the internet.
-a.signalForPeerQueue.setSyncHandler((signal:SignallingMessage) => {
-  log.info('a: sending signal to b.');
-  b.handleSignalMessage(signal);
+a.signalForPeerQueue.setSyncHandler((message:signals.Message) => {
```

Comment:
You've made this `signal:signals.Message` in some places, not in others. I prefer `signal` but we should be consistent.